### DISCUSSION TITLE
Filter "In Theater" movies from Discover - Trending?

### DISCUSSION THREAD

**Discussion by x86txt** [sct/overseerr#3585] [Help]
Hi - I'd like to know if there is a way to filter movies from Trending or Popular that are currently in the theater, or otherwise get a list of popular/trending movies that are not in the theater via another method?

Right now I go to Movies and just try and guess at a start/end release date filter, but it's pretty hit or miss.

Thoughts?

Thanks everyone!

**Comment by heavymp** [comment]
Also would like this option
